src/Game.cpp

```diff
@@ -13,3 +14,17 @@ int Game::score() const
     //needs implementation after made checkValueFrame()
     return 0;
 }
+
+void Game::setIsStrikeOrSpare(unsigned short int position, const std::string& value)
+{
+    auto strike = std::find(value.begin(), value.end(), 'X');
+    auto spare = std::find(value.begin(), value.end(), '/');
+    if (*strike == 'X')
```

Comment:
It can cause undefined behaviour. You should check if iterator is not equal to value.end() before dereferencing it.